gettsim/taxes/eink_st.py

```diff
@@ -3,17 +3,17 @@
 from gettsim.typing import IntSeries
 
 
-def st_kein_kind_freib_tu(
-    zu_verst_eink_kein_kinderfreib_tu: FloatSeries,
+def st_kein_kinderfreib_tu(
```

Comment:
```suggestion
def eink_st_kein_kinderfreib_tu(
```

Reply:
We should do that everywhere. Sorry for realising it only now. 

Please search for `" st_"` and replace by `" _eink_st"` unless the soli is already integrated (probably never the case, but thought I'd mention it).

Reply:
Looking for some variations, that should do the trick. Probably useful to clean up the test data at some point, there are some wild columns in there (`_st_abgeltst_kein_kinderfreib_tu`, `_st_abgelt_st_m_kinderfreib_tu`, ...)